scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/ConsumerDispatch.java

```diff
@@ -0,0 +1,11 @@
+package com.scylladb.cdc.model.worker;
+
+import java.util.concurrent.CompletableFuture;
+
+interface ConsumerDispatch {
```

Comment:
Please add a javadoc describing what is this interface suppose to do.

Reply:
I can't find any usage of ConsumerDispatch in this PR which makes in hard to judge whether it makes sense or not.

Reply:
What do you mean by that? `ConsumerDispatch` is used in `Consumer`:

https://github.com/scylladb/scylla-cdc-java/pull/38/files#diff-e51d1ce424aca5c9e64582a4c67b1e71ed6602f8319dcc307c8e3e9a7d26ae6aR17

Reply:
I mean that I don't see any code actually creating Consumer using ConsumerDispatch.